src/contacts/dao/IRecordDao.java

```diff
@@ -0,0 +1,15 @@
+package contacts.dao;
+
+import contacts.model.Record;
+
+import java.io.IOException;
+import java.util.List;
+
+public interface IRecordDao {
+    Record get(int index);
```

Comment:
it can be probably renamed to getByIndex()

Reply:
Done